src/main/java/com/easypost/net/HashMapSerializer.java

```diff
@@ -0,0 +1,177 @@
+package com.easypost.net;
```

Comment:
Unfortunately not. The `main` namespace isn't aware of the `test` namespace (the `test` namespace knows about the `main` namespace), so I can't host any files that the `main` namespace needs in the `test` namespace.

After working on this, I can actually see the bright side of this work, that being making the JSON serialization more visible. As I note in the code comment, since this is an override for how Gson should serialize any HashMaps, and virtually all data that will need to be serialized for our API calls are HashMaps, this ruleset will handle the majority of all JSON serialization for our application. Previously, this process was blackboxed into Gson's internal `toJsonTree` method, with little control for us to override defaults or troubleshoot invalid serializations. Now that we have effectively overridden that functionality with our own, we have more control over the process if it were to ever break or need to be altered. This also helps us ensure the process is more uniform and standardized.

The C# library underwent a similar renovation a few months ago, where we implemented a custom JSON serializer of sorts, allowing us greater control and access to the process.

The Java library's JSON process has been a sticky operation for some time now (we've had multiple internal tickets in an attempt to fix it or redesign it). I believe this is a step in the right direction.

It has been tested against every API call in our library currently and correctly serialized all data. At the current moment, it seems to cover every case (the HashMap serializer ruleset is quite similar to the [censoring ruleset](https://github.com/EasyPost/easyvcr-java/blob/66c96da9415d8d8889a8dc342deda19e717f22d9/src/main/java/com/easypost/easyvcr/Censors.java#L88) built for the EasyVCR C#/Java applications, if you want more evidence that this implementation is effective).